README.md

```diff
@@ -0,0 +1 @@
+The site is for searching bus schedules. You can save your preferences for later use!

```

Comment:
Should we add a heading? https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers

Maybe even a development sub heading with some work in progress type?

The development section would contain instructions how to start developing this project locally? So its pretty useful for future selfs and generations.